bin/tictactoe

```diff
@@ -1,3 +1,23 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+# puts 'Hello, Human!'
+# puts 'Welcome to Tic Tac Toe!'
+# puts 'How many humans will be playing today?'
+# puts '1? 2? Or would you like to watch the computers battle it out?'
+
+# number_of_humans = gets.strip.to_i
+
+# case number_of_humans
+#   when 1
+#     Game.new(Player::Human.new('X'), Player::Computer.new('O')).play
+
+#   when 2
+#     Game.new(Player::Human.new('X'), Player::Human.new('O')).play
+
+#   when 0
+#     Game.new(Player::Computer.new('X'), Player::Computer.new('O')).play
+# end
+
+Game.start
```

Comment:
Great bin file, short length is ideal. However, we would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.